sunpy/net/dataretriever/sources/goes.py

```diff
@@ -159,7 +173,7 @@ def _makeimap(self):
         """
         Helper function used to hold information about source.
         """
-        self.map_["source"] = "nasa"
+        self.map_["source"] = "noaa"
```

Comment:
Should we keep `NOAA/NASA` or just `NOAA`?

Reply:
good question - GOES XRS is a NOAA mission - however for data before GOES 13 the data we pull is from a NASA server. I think it's probably ok to say just NOAA as its not actually NASA data